doc/boot_and_update.md

```diff
@@ -1,5 +1,132 @@
 # How to boot Mudita Pure and create a storage partition
 
+## Generall upadte architecture
```

Comment:
update

Reply:
tx I'm checking my poor writing skills just now :) (And requesting @SP2FET to help me with that as he always proof reads my documentation :D )